extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,26 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: FSInternetRadio Binding
+Bundle-SymbolicName: org.eclipse.smarthome.binding.fsinternetradio;singleton:=true
+Bundle-Vendor: Eclipse.org/SmartHome
+Bundle-Version: 0.8.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.7
+Bundle-ClassPath: .
+Import-Package: 
+ org.apache.commons.httpclient,
```

Comment:
I understand your rationale, but for now it would require too much time changing and testing the connection code.